astroquery/alma/core.py

```diff
@@ -262,6 +325,12 @@ def stage_data(self, uids):
         .DEBUG: Data list URL: https://almascience.eso.org/rh/requests/anonymous/786823226 [astroquery.alma.core]
         """
 
+        import time
```

Comment:
why have been the imports moved down here?

Reply:
because I want to delete them when we get rid of `stage_data_prefeb2020` - it's a memory aid.  Also, because `stage_data_prefeb2020` is a collection of hacks, and I want to remind ourselves that this collection of hacks should be burnt in a glorious pyre